web/packages/teleterm/src/ui/DocumentsReopen/DocumentsReopen.tsx

```diff
@@ -48,7 +48,7 @@ export function DocumentsReopen(props: {
 
   return (
     <DialogConfirmation
-      open={!history}
```

Comment:
Heh, I'm not sure if there's a single lint rule which would help here. :D Even no-implicit-coercion wouldn't catch this. https://eslint.org/docs/latest/rules/no-implicit-coercion

Reply:
I think https://eslint.org/docs/latest/rules/no-restricted-globals could help 🧐